src/main/java/mil/dds/anet/threads/mart/MartImporterWorker.java

```diff
@@ -0,0 +1,89 @@
+package mil.dds.anet.threads.mart;
+
+import graphql.GraphQLContext;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import microsoft.exchange.webservices.data.core.service.item.EmailMessage;
+import microsoft.exchange.webservices.data.property.complex.FileAttachment;
+import mil.dds.anet.beans.JobHistory;
+import mil.dds.anet.config.AnetDictionary;
+import mil.dds.anet.database.JobHistoryDao;
+import mil.dds.anet.threads.AbstractWorker;
+import mil.dds.anet.threads.mart.services.IMailReceiver;
+import mil.dds.anet.threads.mart.services.IMartReportImporterService;
+import mil.dds.anet.threads.mart.services.IMartTransmissionLogImporterService;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConditionalOnExpression("not ${anet.no-workers:false} and not ${anet.mart.disabled:true}")
+public class MartImporterWorker extends AbstractWorker {
+
+  public static final String REPORT_JSON_ATTACHMENT = "mart_report.json";
+  public static final String TRANSMISSION_LOG_ATTACHMENT = "mart_transmission_log.json";
+
+  private final IMailReceiver mailReceiver;
+  private final IMartReportImporterService martReportImporter;
+  private final IMartTransmissionLogImporterService transmissionLogImporter;
+
+  public MartImporterWorker(AnetDictionary dict, JobHistoryDao jobHistoryDao,
+      IMailReceiver mailReceiver, IMartReportImporterService martReportImporter,
+      IMartTransmissionLogImporterService martTrasmissionLogImporter) {
```

Comment:
Please rename to `transmissionLogImporter` (original parameter had a typo as well).